extensions/s3/src/main/java/io/deephaven/extensions/s3/S3AsyncClientFactory.java

```diff
@@ -41,6 +39,25 @@ class S3AsyncClientFactory {
     private static volatile ScheduledExecutorService scheduledExecutor;
 
     static S3AsyncClient getAsyncClient(@NotNull final S3Instructions instructions) {
+        S3AsyncClient s3AsyncClient;
+        try {
+            s3AsyncClient = getAsyncClientBuilder(instructions).build();
+        } catch (final RuntimeException e) {
```

Comment:
I think we can be more specific here, and only catch `SdkClientException`.

Also, I think we can also be a bit more specific, and match the error message starts with "Unable to load region". A quick check, this error message has been "stable" for the last 6 years. It's an implementation detail we shouldn't necessarily rely on, but I don't see any better way to differentiate at this time.